app/controllers/users_controller.rb

```diff
@@ -1,18 +1,25 @@
 class UsersController < ApplicationController
   def show
     if current_user.token
-      conn = Faraday.new(url: "https://api.github.com") do |f|
-            f.headers['Authorization'] = ("token #{current_user.token}")
-            f.adapter Faraday.default_adapter
+      conn = Faraday.new(url: 'https://api.github.com') do |f|
+        f.headers['Authorization'] = "token #{current_user.token}"
+        f.adapter Faraday.default_adapter
       end
 
-      repo_response = conn.get("/user/repos")
+      repo_response = conn.get('/user/repos')
 
       repo_hash = JSON.parse(repo_response.body, symbolize_names: true)[0..4]
 
       @repos = repo_hash.map do |repo_data|
         Repo.new(repo_data)
       end
+
```

Comment:
Yep, same thing I did for followers. We should take some time to refactor soon so this isn't so long!